#### (1)-
##### The probability distribution of discrete random variables are described by Probability Mass Function (PMF). It is a mathematical function that describes the probability that a random variable will take on a specific value rather than falling within a range of values. Ex) Rolling a fair dice and finding probability for getting a specific number say 5.
##### The probability distribution of continuous random variables are described by Probability Density Function (PDF). It is a mathematical function that describes the probability that a random variable will fall within a certain range of values. Ex) Taking a heights dataset and finding the probability of a height.

#### (2)-
##### Cumulative Density Function (CDF) is used to describe the probability distribution of discrete as well as continuous random variables. It gives the cumulative probability. Ex) If we roll a fair dice then the probability of getting values till 4 will be probability of getting 1 plus probability of getting 2 plus probability of getting 3 plus probability of getting 4= 1/6+1/6+1/6+1/6= 4/6. The Cumulative Distribution Function (CDF) of a real-valued random variable X, evaluated at x, is the probability function that X will take a value less than or equal to x. The cumulative distribution function (CDF) calculates the cumulative probability for a given x-value.

#### (3)-
##### Situations where the normal distribution is used as a model- Heights of people, blood pressure, weights, student's average report etc. 
##### The two parameters of normal distribution are mean and standard deviation. The mean is located at the centre. The mean defines the location of the peak and most data points are clustered around mean. Most of the data of the random variable (following normal distribution) falls within 3 standard deviations from the mean. So, the standard deviation or variance determines the width of the curve. In this way these 2 parameters relate to the shape of the distribution. Using mean and standard deviation and calculating the y values for each data point(x) using gaussian function, the bell curve can be plotted.

#### (4)-
##### Normal/Gaussian distribution is one of the most important probability distributions in statistics because they accuartely describe the distribution of values for many natural phenomenon. Most of the data in universe follow Normal/Gaussian distribution.
##### Real life examples of normal distribution- Birth weight of babies, shoe sizes, Blood pressures, Stock price returns etc.

#### (5)-
##### Bernoulli distribution is the discrete probability distribution of a random variable which has binary outcomes like 1 with probability p and 0 with probability q= 1-p. Informally it can be thought of as a model for any experiment whose outcomes are basically yes/success/1 or no/failure/0.
##### Ex) Tossing a fair coin once and finding the probability of getting a head. So, outcome of getting a head is 1 and not getting a head i.e. getting a tail is 0. Thus binary outcomes, so Bernoulli distribution.
##### Binomial distribution is the discrete probability distribution of the number of successes in a sequence of n independent experiments, each asking a yes-no question and each with its own boolean valued outcome: success(with probability p) and failure(with probability 1-p). Whereas Bernoulli distribution is the discrete probability distribution of a random variable in a single trial.

#### (6)-
##### The dataset follows normal distribution.

In [1]:
mean= 50
std= 10
xi= 60

In [2]:
z_score= (xi-mean)/std
z_score

1.0

##### A z_score of 1 corresponds to the point xi lying 1 standard deviation away from the mean. We want to find the probability of a random observation to be greater than 60. So, we want the area under the curve to the right side of 60.
##### Using this Z_score in the Z_table we get AUC from left upto the point 60 = 0.84134. So, AUC to the right side of 60-

In [3]:
AUC= 1-0.84134
AUC

0.15866000000000002

##### Therefore the probability of a random observation to be greater than 60 is 0.15866

#### (7)-
##### Uniform distribution is a probability distribution where the outcomes are equallly likely to occur i.e they have a constant probability. It is of 2 types- 
##### Continuous uniform distribution- Also called as rectangular distribution, the distribution describes an experiment where there is an arbitrary outcome that lies between certain bounds a(min) and b(max). It describes the distribution of a conttinuous random variable using Probability Density Function.
##### Discrete uniform distribution- It is a symmetric distributuion wherein a finite number of values are equally likely to be observed, every one of n values has equal probability 1/n. It describes the distribution of a discrete random variable using Probability Mass Function.
##### Ex) Rolling a fair dice, the probability of getting 1, 2, 3, 4, 5, 6 , each = 1/6. So, this is a discrete random variable with each data point having equal probability. So, it is an uniform distribution and can be described by PMF.

#### (8)-
##### Z-score is a statistical measurement that describes a value's relationship to the mean of a group of values. Z-score tells how many standard deviations away from the mean a point lies.
##### Importance of Z-score-
##### Z-score helps in standardizing the data i.e. converting all values to one unit. This basically means Z-score helps in changing a Normal distribution with some mean and standard deviation, to a Standard Normal distribution with mean= 0 and standard deviation= 1.
##### This in turn increases the training efficiency of ML models.
##### Z-score helps in calculating how many standard deviations away from the mean a particular point lies.

#### (9)-
##### Central limit theorem says that the sampling distribution of mean will always be normally distributed as long as the sample size is large enough, regardless of whether the population has normal, poisson, binomial or any other distribution. Sampling distribution is nothing but the probability distribution of a statistic for a large number of samples taken from a population.
##### Significance of CLT-
##### Election polling is a great example of how you can use CLT. These polls are used to estimate the number of people who support a specific candidate. The CLT aids in this calculation.
##### We use the CLT in various census fields to calculate various population details such as family income, electricity consumption, individual salaries, and so on.

#### (10)-
##### Assumptions of Central Limit Theorem-
##### The data must adhere to the randomization rule. It needs to be sampled at random.
##### When the population data follows normal distribution, the sample size n can be any value. BUt if the population data doesn't follow normal distribution then the sample size should be >= 30. So, we have to consider large sample size.
##### The samples should be unrelated to one another. One sample should not impact the others.